www/app/hardware/Hardware.html

```diff
@@ -1348,6 +1348,7 @@ <h2><span data-i18n="Hardware">Hardware</span>:</h2>
 						<option value="0xF31B" data-i18n="Distance">Distance</option>
 						<option value="0xF31D" data-i18n="Electric (Instant+Counter)">Electric (Instant+Counter)</option>
 						<option value="0xF321" data-i18n="Managed counter">Managed counter</option>
+						<option value="0xF322" data-i18n="Managed multi counter">Managed multi counter</option>
```

Comment:
Please try to use Camel Case here ... (Also for the previous managed counter)
Managed Counter
Managed Multi Counter

This makes it consistent throughout the application

Reply:
Cf. 1991c520ce9ccc5c88184ee8fefd23e379fd5250